site/en/blog/user-agent-reduction-deprecation-trial/index.md

```diff
@@ -1,22 +1,32 @@
 ---
 layout: "layouts/blog-post.njk"
-title: "User-Agent Reduction deprecation trial"
+title: "User-Agent Reduction deprecation trial has now ended"
```

Comment:
Is our plan to update all blogs after the fact? I thought blogs were very much a snapshot of the moment, and the assumption was that they would get outdated with time?

Reply:
I had that exact question. It's up to you all.

Reply:
reverted text changes. added aside about being outdated.